adbe/adb_enhanced.py

```diff
@@ -1493,10 +1493,13 @@ def perform_install(file_path):
         print_error('Failed to install %s, stderr: %s' % (file_path, stderr))
 
 
-@ensure_package_exists
-def perform_uninstall(app_name):
+@ensure_package_exists2
+def perform_uninstall(app_name, user_zero):
```

Comment:
user_zero -> first_user or something more illustrative of `what` and not `how`?

Reply:
`privileged` ?

Reply:
Any news regarding this PR ?  Do you confirm that --first-user is the argument to use ?